sections/dashboard/Overview/Overview.tsx

```diff
@@ -46,6 +47,7 @@ const Overview: FC = () => {
 				name: PositionsTab.CROSS_MARGIN,
 				label: t('dashboard.overview.positions-tabs.cross-margin'),
 				badge: crossPositions,
+				titleIcon: <FuturesIcon type="cross_margin" />,
```

Comment:
This is wrapping now on my screen, 14" macbook window maximized:
<img width="370" alt="image" src="https://user-images.githubusercontent.com/10401554/195133075-24d6fa28-27d1-480c-a086-2be0b5d91f77.png">